package.json

```diff
@@ -1,29 +1,36 @@
 {
+  "type": "module",
```

Comment:
**Question**: Is it needed? If I run `build` I get
> error esbuild error
    require() of ES Module /Users/noriste/Sites/noriste/figma-to-xstate/build-figma-plugin.main.js from /Users/noriste/Sites/noriste/figma-to-xstate/node_modules/.pnpm/import-fresh@3.3.0/node_modules/import-fresh/index.js not supported.
    build-figma-plugin.main.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
    Instead rename build-figma-plugin.main.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /Users/noriste/Sites/noriste/figma-to-xstate/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

Reply:
Yes, it's a recommendation from the eslint plugin. And it's better to keep things ESM.